src/internal/connector/exchange/api/api.go

```diff
@@ -57,37 +59,51 @@ type Client struct {
 	// The stable service is re-usable for any non-paged request.
 	// This allows us to maintain performance across async requests.
 	stable graph.Servicer
+
+	// The largeItem graph servicer is configured specifically for
+	// downloading large items.  Specificlaly for use when handling
+	// attachments, and for no other use.
+	largeItem graph.Servicer
 }
 
 // NewClient produces a new exchange api client.  Must be used in
 // place of creating an ad-hoc client struct.
 func NewClient(creds account.M365Config) (Client, error) {
-	s, err := newService(creds)
+	s, li, err := newServices(creds)
 	if err != nil {
 		return Client{}, err
 	}
 
-	return Client{creds, s}, nil
+	return Client{creds, s, li}, nil
 }
 
 // service generates a new service.  Used for paged and other long-running
 // requests instead of the client's stable service, so that in-flight state
 // within the adapter doesn't get clobbered
 func (c Client) service() (*graph.Service, error) {
-	return newService(c.Credentials)
+	s, _, err := newServices(c.Credentials)
```

Comment:
I think this should have the first part of `newServices` and `newServices` should call this function instead of the other way around. It's usually better to build larger things from smaller things rather than getting smaller things by throwing out part of larger things. Plus, then there won't be any questions as to whether or not the HTTP clients that we create and then discard in this function are consuming resources

Reply:
Sure, I'm good with that.